epsagon/lambda_wrapper.go

```diff
@@ -121,25 +142,47 @@ func (handler *epsagonLambdaWrapper) Invoke(ctx context.Context, payload json.Ra
 		if !handler.invoked {
 			result, err = handler.handler(ctx, payload)
 		}
+		if invokeInfo.thrownError != nil {
+			panic(invokeInfo.thrownError)
```

Comment:
I don't think this is good, it will show the error is thrown from here instead of the users function. If we don't call `recover()` than it will keep throwing this up and the stack will show from there.